CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -471,7 +496,7 @@ private void NoInliningConstructor()
 
             PresentationSource.AddSourceChangedHandler(this, PresentationSourceChangedHandler);
 
-            RenderOptions.SetBitmapScalingMode(this, BitmapScalingMode.HighQuality);
+            RenderOptions.SetBitmapScalingMode(this, BitmapScalingMode.NearestNeighbor);
```

Comment:
Revert this, you can change it in your own code if you wish to adjust the quality.